src/sweetalert2.js

```diff
@@ -1,1253 +1,1232 @@
-        $confirmButton.style.borderRightColor = params.confirmButtonColor;
+      var btnIndex = -1; // Find the button - note, this is a nodelist, not an array.
+      for (var i = 0; i < $modalElements.length; i++) {
+        if ($targetElement === $modalElements[i]) {
+          btnIndex = i;
+          break;
+        }
       }
 
-      sweetAlert.enableLoading = function() {
-        addClass($confirmButton, 'loading');
-        addClass(modal, 'loading');
-        $cancelButton.disabled = true;
-      };
-
-      sweetAlert.disableLoading = function() {
-        removeClass($confirmButton, 'loading');
-        removeClass(modal, 'loading');
-        $cancelButton.disabled = false;
-      };
-
-      sweetAlert.enableButtons = function() {
-        $confirmButton.disabled = false;
-        $cancelButton.disabled = false;
-      };
-
-      sweetAlert.disableButtons = function() {
-        $confirmButton.disabled = true;
-        $cancelButton.disabled = true;
-      };
-
-      sweetAlert.enableInput = function() {
-        var input = getInput();
-        if (input.type === 'radio') {
-          var radiosContainer = input.parentNode.parentNode;
-          var radios = radiosContainer.querySelectorAll('input');
-          for (var i = 0; i < radios.length; i++) {
-            radios[i].disabled = false;
-          }
+      // TAB
+      if (keyCode === 9) {
+        if (!e.shiftKey) {
+          // Cycle to the next button
+          setFocus(btnIndex, 1);
         } else {
-          input.disabled = false;
+          // Cycle to the prev button
+          setFocus(btnIndex, -1);
         }
-      };
-
-      sweetAlert.disableInput = function() {
-        var input = getInput();
-        if (input.type === 'radio') {
-          var radiosContainer = input.parentNode.parentNode;
-          var radios = radiosContainer.querySelectorAll('input');
-          for (var i = 0; i < radios.length; i++) {
-            radios[i].disabled = true;
-          }
-        } else {
-          input.disabled = true;
-        }
-      };
-
-      sweetAlert.showValidationError = function(error) {
-        var $validationError = modal.querySelector('.' + swalClasses.validationerror);
-        $validationError.innerHTML = error;
-        show($validationError);
-
-        var input = getInput();
-        focusInput(input);
-        addClass(input, 'error');
-      };
-
-      sweetAlert.resetValidationError = function() {
-        var $validationError = modal.querySelector('.' + swalClasses.validationerror);
-        hide($validationError);
-
-        var input = getInput();
-        removeClass(input, 'error');
-      };
-
-      sweetAlert.enableButtons();
-      sweetAlert.disableLoading();
-      sweetAlert.resetValidationError();
-
-      window.onfocus = function() {
-        // When the user has focused away and focused back from the whole window.
-        window.setTimeout(function() {
-          // Put in a timeout to jump out of the event sequenc
```

Comment:
When I'm adding new features of fixing bugs, I'm using [index.html](https://github.com/limonte/sweetalert2/blob/master/index.html) to test swal2.

For dev env I'm changing [line 18](https://github.com/limonte/sweetalert2/blob/master/index.html#L18) to `src/sweetalert2.js` so I can easy debug unminified source code.

With this change I'm getting `sweetalert2.js:1232 Uncaught SyntaxError: Unexpected token export`. I want to be able to run and test index.html with unminified version.